relay/kusama/src/lib.rs

```diff
@@ -1860,7 +1860,7 @@ pub mod migrations {
 	/// Upgrade Session keys to exclude `ImOnline` key.
 	/// When this is removed, should also remove `OldSessionKeys`.
 	pub struct UpgradeSessionKeys;
-	const UPGRADE_SESSION_KEYS_FROM_SPEC: u32 = 1001002;
+	const UPGRADE_SESSION_KEYS_FROM_SPEC: u32 = 1001003;
```

Comment:
It would be safer to create a storage key like `:upgrade_session_keys_migration_executed` and then manually cleaning it up via governance.  
But i think we are past that point.